dev/Repeater/UniformGridLayoutFactory.cpp

```diff
@@ -90,6 +91,18 @@ void UniformGridLayout::EnsureProperties()
                 box_value(winrt::UniformGridLayoutItemsJustification::Start), /* defaultValue */
                 winrt::PropertyChangedCallback(&UniformGridLayout::OnPropertyChanged));
     }
+
```

Comment:
Is there a reason that this isn't generated code? Is there something the codegen tool doesn't support?

Reply:
It looks like it is being generated. Can we get rid of this code and just use the generated code?